opportunity/README.md

```diff
@@ -8,77 +8,215 @@ The STAPI Opportunity describes a single business unit available for ordering.
-resources below.
+Additional metadata fields can be added to the GeoJSON Object Properties that
+describe the Opportunity in more detail for the user. The only required fields
+are `datetime` and `product_id but it is recommended to add more fields as
+required to describe the opportunity in meaningful terms to the requestor.
 
-| Field Name | Type         | Description                                                  |
-| ---------- | ------------ | ------------------------------------------------------------ |
+| Field Name | Type | Description |
+| ---------- | ---- | ----------- |
 | datetime       | string                                                                     | **REQUIRED.** Datetime field is a [ISO8601 Time Interval](https://en.wikipedia.org/wiki/ISO_8601#Time_intervals) |
 | product_id | string | **REQUIRED.**  Product identifier. The ID should be unique and is a reference to the [parameters](https://github.com/Element84/stapi-spec/blob/main/product/README.md#parameters) which can be used in the [parameters](https://github.com/Element84/stapi-spec/blob/main/product/README.md#parameters) field. |
 
-#### Link Object
+#### Opportunity Links
 
-Each link in the links array must be a [Link](https://github.com/radiantearth/stac-spec/blob/master/commons/links.md#link-object) Object.
+Each link in the links array must be a
+[Link](https://github.com/radiantearth/stac-spec/blob/master/commons/links.md#link-object)
+Object.
 
 ##### rel=create-order
 
-This Link object fully describes the necessary HTTP request to submit an Order for this Opportunity via [Create Order](https://github.com/stapi-spec/stapi-spec/tree/main/order#create-order-request).
+This Link object fully describes the necessary HTTP request to submit an Order
+for this Opportunity via
+[Create Order](https://github.com/stapi-spec/stapi-spec/tree/main/order#create-order-request).
 
 To conform to the Create Order spec, use `"method": "POST"`.
 
 If no Body parameters apply to an Opportunity, use `"body": {}`.
+
+An Opportunity's links **must** include a `rel=create-order` link that allows
+the user to Order this opportunity.
+
+#####
+
+## Async Opportunity Search
+
+- **Conformance URI:** <https://stapi.example.com/v0.1.0/async-opportunities>
+
+STAPI has an optional conformance class providing support for async opportunity
+searches, to accommodate searches for products that require more time to
+complete than is a typical or desirable HTTP request duration. This async
+support requires persisting the search state and matching opportunities outside
+the context of the initial request, so they are available for retrival at a
+later time by the requestor.  The Opportunity Search Record is an additional
+entity defined to model the required search state.
+
+### Opportunity Search Record
```

Comment:
Need to also include the search parameters themselves as part of the search object.